libs/auth-api-lib/src/lib/resources/utils/col.ts

```diff
@@ -0,0 +1,26 @@
+import snakeCase from 'lodash/snakeCase'
+
+const JOIN_SEPARATOR = '->'
+const COL_SEPARATOR = '.'
+
+/**
+ * Returns root-level column references like this:
+ * $association->nestedAssociation.column_name$
+ * @param args
+ */
+export function col(...args: [...Array<string | undefined>, string]): string {
```

Comment:
Really nice utility to abstract the column name creation. Wondering if we should put it rather in our `@island.is/nest/sequelize` library to make it more global utility?

Reply:
I'll create a task. Need to get this PR in asap :)